### PR TITLE
KK-250 | Ensure that login page is in user's chosen language

### LINKED ISSUE #146: Login page in Finnish when using English or Swedish site
1. Start from English (or Swedish) site
2. Enter details and click "Register"

image

Expected result: login page is English (or Swedish)

Actual result: login page is Finnish

image

### PR DESCRIPTION
**Pull request by badrange**
This should fix #146 - it does require that Tunnistamo is configured correctly.

### CODE REVIEW

File: src/domain/auth/authenticate.ts [hunk 1]
```diff
@@ -12,12 +12,15 @@ import {
...
+import { getCurrentLanguage } from '../../common/translation/TranslationUtils';
 
 export const loginTunnistamo = (path?: string) => {
   userManager
-    .signinRedirect(
-      path ? { data: { path: path } } : { data: { path: '/profile' } }
-    )
+    .signinRedirect({
+      data: { path: path || 'profile' },
+      /* eslint-disable @typescript-eslint/camelcase */
```
Comment (midori0507) [inline src/domain/auth/authenticate.ts]: Imported I18n instance doesnt reflect user selected language correctly.
Only i18n from component is.

Comment (badrange) [reply src/domain/auth/authenticate.ts]: We can only use i18n from component if we are working with a component? I see no FC component around here?

I tested my implementation, and it works correctly also when changing language.

Comment (midori0507) [reply src/domain/auth/authenticate.ts]: Yeah just worry if you need exact correct language, lgtm

Comment (badrange) [reply src/domain/auth/authenticate.ts]: One more possible corner case bug in here :D 
But it is at least in the same file. And the risk is low and impact negligible.

**Pull request merged**